front-end/src/app/shared/components/table-list-base/table-list-base.component.ts

```diff
@@ -167,4 +168,9 @@ export abstract class TableListBaseComponent<T> implements OnInit, AfterViewInit
       this.messageService.add({ severity: 'success', summary: 'Successful', detail: 'Items Deleted', life: 3000 });
     });
   }
+
+  public getGetParams(): { [param: string]: string | number | boolean | ReadonlyArray<string | number | boolean> } {
```

Comment:
It's not obvious what why this method is here, please add a comment using [JSDocs](https://devhints.io/jsdoc). See OnSelectAllChange() method above for an example

Reply:
Commenting added per the request.  It's definitely a little obtuse, so this was a necessary change.